+++ src/lib/gtest/src/gtest.cc
@@ -2876,17 +2876,7 @@
     // console there does support colors.
     return stdout_is_tty;
 #else
-    // On non-Windows platforms, we rely on the TERM variable.
-    const char* const term = posix::GetEnv("TERM");
-    const bool term_supports_color =
-        String::CStringEquals(term, "xterm") ||
-        String::CStringEquals(term, "xterm-color") ||
-        String::CStringEquals(term, "xterm-256color") ||
-        String::CStringEquals(term, "screen") ||
-        String::CStringEquals(term, "screen-256color") ||
-        String::CStringEquals(term, "linux") ||
-        String::CStringEquals(term, "cygwin");
-    return stdout_is_tty && term_supports_color;
+    return stdout_is_tty;
 #endif  // GTEST_OS_WINDOWS
   }
 
